testing/php7-pdlib/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: goodspb <goodspb.luo@gmail.com>
+# Maintainer: Adriaan Groenenboom <agboom@pm.me>
+pkgname=php7-pdlib
+_pkgreal=pdlib
+pkgver=1.0
+pkgrel=0
+pkgdesc="PHP extension for Dlib"
+url="https://github.com/goodspb/pdlib"
+arch="all"
+license="MIT"
+depends="php7-common dlib"
```

Comment:
dlib could be moved to makedepends because dependencies are tracked by abuild
```
>>> php7-pdlib*: Tracing dependencies...
	php7-common
	dlib
	so:libc.musl-x86_64.so.1
	so:libdlib.so.19.18.0
	so:libgcc_s.so.1
	so:libstdc++.so.6
```

Reply:
What does it mean that dependencies are tracked by abuild? Does abuild automatically detect dependencies?

Reply:
Removing `depends="php7-common dlib"` shows the following error on `abuild -r`:

```
checking for pkg-config... configure: error: pkg-config not found # <--- means it cannot find dlib, the error is a bit obscure
>>> ERROR: php7-pdlib: build failed
```

Reply:
Probably it can't find for different reason but ok for now
Please squash commits `git rebase -i HEAD~2` and set commit message to `testing/php7-pdlib: new aport
`

Reply:
Ok done